pkg/helpers/bulk/cmd.go

```diff
@@ -17,11 +18,10 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/dynamic"
-	kapi "k8s.io/kubernetes/pkg/apis/core"
 )
 
 type Runner interface {
-	Run(list *kapi.List, namespace string) []error
+	Run(list *metainternalversion.List, namespace string) []error
```

Comment:
:scream: every time I look at it it's a mess, we should probably invest a bit of time and clean it thoroughly.  Not in this PR, though :wink:

Reply:
ok, i'll put that on my list of TODOs :)